net/wallet/Portfile

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+perl5.branches      5.24
+
+name                wallet
+version             1.3
+revision            0
+categories          net security
+license             MIT
+maintainers         kornel.us:karl macports.org:openmaintainer
```

Comment:
`openmaintainer` is used without the `macports.org:` prefix; it's a keyword that has special handling in `port(1)`.